src/Container/index.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import RBContainer, { type ContainerProps as RBContainerProps } from 'react-bootstrap/Container';
+
+import type { ComponentWithAsProp } from '../utils/types/bootstrap';
+
+enum ContainerSizeClass {
+  xs = 'container-mw-xs',
+  sm = 'container-mw-sm',
+  md = 'container-mw-md',
+  lg = 'container-mw-lg',
+  xl = 'container-mw-xl',
+}
+
+export type ContainerSize = keyof typeof ContainerSizeClass;
+
+interface ContainerProps extends RBContainerProps {
+  size?: ContainerSize;
+}
+
+type ContainerType = ComponentWithAsProp<'div', ContainerProps> & { Deprecated?: any };
+
+const Container: ContainerType = React.forwardRef<HTMLDivElement, ContainerProps>(({
```

Comment:
Could `Container` be used with a non-`div` element, e.g. with `as="span"`? I wonder if this should be more generic, e.g. `HTMLElement` or even `Element`.

Reply:
Yes! I changed it to `Element` as our property of interest (`className`) exists.
Here: 7e0fb5c8fffafed9b8fefa434a0306263a9312a4